revival/willo/index.html

```diff
@@ -72,16 +72,6 @@ <h2>Jacob Witt</h2>
               alt="" />
         <img  src="../../assets/img/willo/willo-08.jpg" 
               alt="" />
```

Comment:
As far as I can tell the image `willo-8.jpg` should be included on the page.
Screenshot from my branch showing the image:
<img width="600" alt="Screenshot 2023-06-23 at 4 55 37 PM" src="https://github.com/norawarschewski/tw-2023/assets/10406456/4a1da198-a47f-47d8-aab3-2e49ea0e7e4a">

It looks like you removed it yesterday and I didn't pull from main before making this PR so now we have a little merge conflict.